frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1056,8 +1126,7 @@ function ReaderFooter:addToMainMenu(menu_items)
                         end,
                         callback = function()
                             self.settings.item_prefix = "letters"
-                            self:updateFooter()
-                            UIManager:setDirty(nil, "ui")
+                            self:refreshFooter(true)
```

Comment:
> Refresh the footer instantly when changing the prefix, separator, or formatting.
(Either that never worked, or it's an old-ish regression).

I noticed that too - I guess it was just forgotten when making the other stuff changes refresh the footer (migrating them to use refreshFooter())